deploy/pmm-server-minikube-ephemeral.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: pmm
+spec:
+  selector:
+    app: pmm
+  type: NodePort
+  ports:
+  - port: 80
+    name: web
+    targetPort: 80
+    nodePort: 30080
+  - port: 443
+    name: api
+    targetPort: 443
+    nodePort: 30443
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+    name: pmm-net
+spec:
+  selector:
+     app.kubernetes.io/part-of: pmm
+  ports:
+    - name: pmm-server
+      port: 443
+    - name: vm-agent
+      port: 8428
+
+---
+
+apiVersion: v1
```

Comment:
where is this config map used?

Reply:
nowhere here, but could be used by client:
https://github.com/percona/community/pull/216